src/manifestoo/addon_sorter.py

```diff
@@ -0,0 +1,31 @@
+from typing import Dict, Iterable, Set
+
+from graphlib import TopologicalSorter
+
+from manifestoo_core.addons_set import AddonsSet
+
+
+class AddonSorter:
+    def sort(
+        self, addons_selection: Iterable[str], addon_set: AddonsSet
+    ) -> Iterable[str]:
+        pass
+
```

Comment:
Add a class method here. `def from_name(cls, name: str) -> "AddonSorter"`

Reply:
This method can print the error and raise typer.Exit() if the name is unknown or the python version does not have graphlib.

Reply:
Not sure how to handle the (un)availability of graphlib here :thinking: 

Shouldn't be managed by a try except on the import?

Reply:
You can import `graphlib` in the function. Not super pretty but good enough I think.